katello/katello.spec

```diff
@@ -142,16 +142,17 @@ Useful utilities for debug info collecting
 %files debug
 %{_datadir}/foreman/script/foreman-debug.d/katello-debug.sh
 
-%package capsule
+%package foreman-proxy-content
```

Comment:
I think without the katello prefix is the better option (so use the `-n` option)